templates/includes/sidebar.html

```diff
@@ -1,4 +1,9 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <p>User: {{ user.first_name }}</p>
+    <li><a href="{% url 'logout' %}">Logout</a></li>
+  {% endif %}
+  <li><a href="{% url 'login' %}">Login</a></li>
```

Comment:
The 'Login' link should be displayed only when the user is not authenticated. Consider wrapping it with an `{% else %}` block after the `{% if user.is_authenticated %}` block.